src/modelio.cs

```diff
@@ -986,13 +1007,31 @@ interface MDLNormalMapTexture
 	[BaseType (typeof(NSObject))]
 	interface MDLObject : MDLNamed
 	{
+		[iOS (10,3), TV (10,2), Mac (10,12,4)]
+		[Export ("components", ArgumentSemantic.Copy)]
+		IMDLComponent[] Components { get; }
+
+#if XAMCORE_4_0
+		[Internal]
+#endif
+		[Obsolete ("Use SetComponent (Type protocol)")]
 		[Export ("setComponent:forProtocol:")]
 		void SetComponent (IMDLComponent component, Protocol protocol);
 
+		[Wrap ("SetComponent (component, new Protocol (type))")]
+		void SetComponent (IMDLComponent component, Type type);
+
+#if XAMCORE_4_0
+		[Internal]
+#endif
+		[Obsolete ("Use GetComponent (Type protocol)")]
```

Comment:
Obsolete `IsComponentConforming` but expose `GetComponent (Protocol protocol)` for the same reasons as above